engine/src/test/java/io/camunda/zeebe/engine/processing/streamprocessor/StreamProcessorReplayTest.java

```diff
@@ -108,15 +97,7 @@ public void shouldSkipRejections() {
     startStreamProcessor(typedRecordProcessor, eventApplier);
 
     // then
-    final InOrder inOrder = inOrder(typedRecordProcessor, eventApplier);
-    inOrder
-        .verify(typedRecordProcessor, never())
-        .processRecord(anyLong(), any(), any(), any(), any());
-    inOrder.verify(eventApplier, never()).applyState(anyLong(), eq(ACTIVATE_ELEMENT), any());
-    inOrder
-        .verify(typedRecordProcessor, TIMEOUT.times(EXPECTED_ON_RECOVERED_INVOCATIONS))
-        .onRecovered(any());
-    inOrder.verifyNoMoreInteractions();
+    verifyNoInteractions(typedRecordProcessor, eventApplier);
```

Comment:
Wondering how reliable this test really is, since we don't know whether the processing is done right? So it is here hard to test that nothing happens.